.github/workflows/release_gem.yml

```diff
@@ -0,0 +1,14 @@
+name: Release
```

Comment:
I wonder why the github actions are never prepended with `---`  ?

Reply:
people being lazy? :)

Reply:
I propose to add another action as done here, then... ;) https://github.com/theforeman/smart_proxy_salt/blob/master/.github/workflows/yamllint.yaml

Reply:
> people being lazy? :)

Yes.